webknossos/webknossos/dataset/view.py

```diff
@@ -844,6 +861,28 @@ def __exit__(
     def __repr__(self) -> str:
         return repr(f"View({self._path}, bounding_box={self.bounding_box})")
 
+    def _check_chunk_size(
+        self, chunk_size: Vec3Int, write_operation: bool = False
```

Comment:
Is there a strong reason for having a default value for `write_operation`? And `False` seems dangerous to me, since this means that the loose case is used by default. I'd make it an non-optional parameter to avoid wrong usages.